src/server/api/routers/user.ts

```diff
@@ -26,4 +26,15 @@ export const userRouter = createTRPCRouter({
   getAll: protectedProcedure.query(async () => {
     return await db.user.findMany();
   }),
+
+  getByRole: protectedProcedure.input(z.string()).query(async ({ input }) => {
```

Comment:
It's a bit of a nit, but could we make this input an object with one field named role? Single object argument functions are pretty common with JS/TS and this function might want to take more arguments in the future (search, offset, limit)